src/main/scala/com/databricks/sparkdl/ImageUtils.scala

```diff
@@ -16,9 +16,11 @@
 
 package com.databricks.sparkdl
 
+import java.awt
 import java.awt.image.BufferedImage
 import java.awt.{Color, Image}
 
+import com.sun.javafx.iio.ImageStorage.ImageType
 import org.apache.spark.ml.image.ImageSchema
 import org.apache.spark.sql.Row
 import org.apache.spark.sql.expressions.UserDefinedFunction
```

Comment:
Lets drop the unused imports here:

`awt`, `sql.function`, `sql. expressions` and `com.sum`.